_data/developer.yml

```diff
@@ -423,6 +423,18 @@ websites:
   btc: true
   othercrypto: true
   doc: https://ultimez.com/blog/ultimez-now-accepts-cryptocurrency-payment-web-services/
+- name: Underlabs Inc.
+  url: https://underlabs.ca
+  img: underlabs.png
+  twitter: underlabs
+  email_address: hello@underlabs.ca
+  region: na
```

Comment:
Are they really only developing for clients from North America? If they are developing for any client, remove the tag.